.github/workflows/tests.yml

```diff
@@ -31,8 +31,8 @@ jobs:
         export PATH=/usr/bin:/bin:/usr/local/bin
         ruby uninstall -d >/dev/null
         ruby uninstall -f >/dev/null
-        ruby install --force-curl
+        bash install.sh
```

Comment:
This step is now duplicated.

Reply:
Fixed by 2bf5f50bb1b861a328736c3bb5c01885c91f93e3.